watch/watch.1

```diff
@@ -7,41 +7,54 @@
 .SH NAME
 watch \- Keep an eye on a command output.
 .SH SYNOPSIS
-\fBwatch\fR \fB\-n\fR [\fIseconds\fR] [\fB\-bt\fR] [\fBcommand\fR [\fIargs\fR...]]
+\fBwatch\fR [\fB\-n\fR \fIseconds\fR] [\fB\-bt\fR] \fBcommand\fR [\fIargs\fR...]
 .SH DESCRIPTION
 The
 .B watch 
-utility executes a command repeatedly every "user-defined" seconds.
-This allows you to watch a program output change over the time.
+utility executes a command repeatedly every
+"user-defined" seconds. This allows you to watch
+a program output change over the time.
 .PP
 It accepts the following options:
 .TP
 .B \-n
-Specifies the interval to refresh the output. Per default, the
-interval is of 2 seconds.
+Specifies the interval to refresh the output.
+Per default, the interval is of 2 seconds.
 .TP
 .B \-b
-Beeps if the program being run returns a non-zero exit status.
+Beeps if the program being run returns a non-zero
+exit status.
 .TP
 .B \-t
-Turns off the information header, letting just the verbatim program
-output being refreshed every time on the screen \(em equivalent to a \fIksh\fR(1)
+Turns off the information header, letting just the
+verbatim program output being refreshed every time
+on the screen \(em equivalent to a \fIksh\fR(1)
 \fIfor ((;;))\fR accompanied by \fIsleep\fR(1).
 .br
-Not recommended, unless you're using \fItmux\fR(1) or another terminal multiplexer
-that shows you information that would originally be shown by that header.
+Not recommended, unless you're using \fItmux\fR(1)
+or another terminal multiplexer that shows you
+information that would originally be shown by that
```

Comment:
I'm not sure referring to the reader is a good idea for manual-style documentation, as it's a less formal tone in English.

Reply:
My Portuguese teacher often said that to me regarding essays, I think that's a bit of a trace of mine in any text that I do write.
Anyway, much obliged for bringing my attention to this, I will try to not talk with the reader when writing the next time.